app/src/main/java/org/desarrolladorslp/teckersapp/ui/messages/MessageAdapter.kt

```diff
@@ -67,18 +76,16 @@ data class MessageAdapter(private val messagesHeader: ArrayList<MessageHeader>)
                     .fit()
                     .into(view.senderImage)
 
-            }
-            catch (e: Exception) {
+            } catch (e: Exception) {
                 Log.d("MessageImageException", e.message)
             }
 
             view.sender.text = messageheader.sender
             view.subject.text = messageheader.subject
 
-            val simpleDateFormat = SimpleDateFormat("dd/MM/yyyy HH:mm")
-            simpleDateFormat.timeZone = TimeZone.getTimeZone("UTC")
-            val myDate = simpleDateFormat.parse(messageheader.timestamp)
-            view.timestamp.text = myDate.toString()
+            view.timestamp.text = ZonedDateTime.parse(messageheader.timestamp).format(
+                DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT)
+            )
```

Comment:
This should be done in another way because it will limit the Android version support